org.msweet.htmldoc.yml

```diff
@@ -0,0 +1,36 @@
+app-id: org.msweet.htmldoc
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: htmldoc
+rename-icon: htmldoc.png
+rename-desktop-file: htmldoc.desktop
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
```

Comment:
fltk doesn't support wayland, does it?

Reply:
1.4 will, 1.3 doesn't.

Reply:
I assume I should comment out the wayland stuff for now???

Reply:
that's the idea. if it doesn't need it it shouldn't be.